templates/00_struct.go.tpl

```diff
@@ -23,6 +23,31 @@ type {{$alias.UpSingular}} struct {
 	{{end -}}
 }
 
+
+{{if .Table.FKeys -}}
```

Comment:
I'm particularly curious what this is all for. Can we not use the .R struct to hold all of this? Or is this some avoidance of an issue where you could have a row that has 3 entities in it, 1 of which is null?

Reply:
you're right, I think we could just add the `boil` tags to the relevant items in the .R struct and use it directly. While that would be a bit of a change to the existing struct, and I took a perhaps too-conservative approach of not wanting to change it, in hindsight it would probably be cleaner.